rbx_reflector/src/cli/defaults_place.rs

```diff
@@ -85,7 +87,36 @@ fn save_place_in_studio(path: &PathBuf) -> anyhow::Result<StudioInfo> {
         }
     }
 
+    #[cfg(target_os = "macos")]
+    {
+        Command::new("osascript")
+				.args([
+					"-e",
+					"tell application \"System Events\"
+                        repeat with theProcess in processes whose name is \"RobloxStudio\"
+                                tell theProcess
+                                    set windowList to windows whose name contains \"defaults-place.rbxlx - Roblox Studio\"
+                                    
+                                    if (count of windowList) > 0 then
+                                        set frontmost to true
+                                        perform action \"AXRaise\" of window 1
+                                    end if
+                                end tell
+                        end repeat
+                    end tell",
+				])
+				.output()?;
+
+        Command::new("osascript")
+            .args([
+                "-e",
+                "tell app \"System Events\" to key code 1 using command down",
+            ])
+            .output()?;
+    }
+
     #[cfg(not(target_os = "windows"))]
+    #[cfg(not(target_os = "macos"))]
```

Comment:
We can use the `any` predicate to express this in one line:
```suggestion
    #[cfg(not(any(target_os = "windows", target_os = "macos")))]
```